static/js/click_handlers.js

```diff
@@ -44,6 +44,12 @@ import * as unread_ops from "./unread_ops";
 import * as util from "./util";
 
 export function initialize() {
+    // LOGIN BUTTON
+
+    $("body").on("click", ".login_button", () => {
+        window.location.href = "/login/";
+    });
```

Comment:
I'm not sure we need click handlers in this file for login_button or signup_button; it seems like they could just be `a href=/login/` (with a conditional in the template for `page_params.development_environment` if needed).

Reply:
In any case, it's safest to have all of our click handlers have `preventDefault` and `stopPropagation` unless we have a specific reason not to.

Reply:
Our `next` addition  to the url means we can't have `a` links now.

Reply:
hmm, I think we can do that as long as we pass the URLs into the HTML template, right?

Reply:
I ran into an issue - https://chat.zulip.org/#narrow/stream/49-development-help/topic/CSRF.20token.20is.20missing.20or.20incorrect

Reply:
Updated with requested changes.